apps/src/lib/levelbuilder/lesson-editor/resourcesEditorRedux.js

```diff
@@ -10,23 +10,27 @@ const REMOVE_RESOURCE = 'resourcesEditor/REMOVE_RESOURCE';
 // Contains operations that can be done for migrated resources,
 // i.e. ones backed by the Resource model on Rails
 
-export const initResources = resources => ({
+export const initResources = (resourceContext, resources) => ({
   type: INIT,
+  resourceContext,
   resources
 });
 
-export const addResource = newResource => ({
+export const addResource = (resourceContext, newResource) => ({
```

Comment:
I chose `resourceContext` because resources can have types (eg slides, activity guide, etc) and I didn't want to overload the word.